bin/trigger

```diff
@@ -25,14 +30,21 @@ and then tell curl to use it:
 which will then not require you to type your password every time.
 
 ¹ https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
-² https://ec.haxx.se/usingcurl/usingcurl-netrc
+² https://docs.github.com/en/rest/overview/other-authentication-methods#via-username-and-password
+³ https://ec.haxx.se/usingcurl/usingcurl-netrc
 .
     exit 1
 fi
 
+auth=':'
+if [[ $TOKEN ]]; then
```

Comment:
Should be `-n "$TOKEN"` I think.

Reply:
Yup! Updated in [46962f8](https://github.com/nextstrain/ncov-ingest/pull/299/commits/46962f8d0f37d0b4e93ab03811dfd61bf3056e7c)